enigma/src/main/java/cuchaz/enigma/translation/mapping/serde/enigma/EnigmaMappingsReader.java

```diff
@@ -77,7 +77,38 @@
 		}
 	};
 
-	protected void readFile(Path path, EntryTree<EntryMapping> mappings) throws IOException, MappingParseException {
+	/**
+	 * Reads multiple Enigma mapping files.
+	 *
+	 * @param progress the progress listener
+	 * @param paths    the Enigma files to read; cannot be empty
+	 * @return the parsed mappings
+	 * @throws MappingParseException    if a mapping file cannot be parsed
+	 * @throws IOException              if an IO error occurs
+	 * @throws IllegalArgumentException if there are no paths to read
+	 */
+	public static EntryTree<EntryMapping> readFiles(ProgressListener progress, Path... paths) throws MappingParseException, IOException {
+		EntryTree<EntryMapping> mappings = new HashEntryTree<>();
+
+		if (paths.length == 0) {
+			throw new IllegalArgumentException("No paths to read mappings from");
+		}
+
+		progress.init(paths.length, I18n.translate("progress.mappings.enigma_directory.loading"));
+		int step = 0;
+
+		for (Path file : paths) {
+			progress.step(step++, paths.toString());
+			if (Files.isHidden(file)) {
```

Comment:
I don't think files should be skipped from reading here since you explicitly pass these as a parameter. What was the reasoning for this?

Reply:
Copy-paste, really. I agree with you here and I'll change it at some point.